depthai_examples/ros1_src/yolov4_spatial_publisher.cpp

```diff
@@ -28,9 +29,13 @@ const std::vector<std::string> label_map = {"person",         "bicycle",    "car
              "laptop",         "mouse",      "remote",        "keyboard",      "cell phone",  "microwave",     "oven",
              "toaster",        "sink",       "refrigerator",  "book",          "clock",       "vase",          "scissors",
              "teddy bear",     "hair drier", "toothbrush"};
+std::unique_ptr<dai::rosBridge::BridgePublisher<sensor_msgs::Image, dai::ImgFrame>> rgbPublish, depthPublish;
+std::unique_ptr<dai::rosBridge::ImageConverter> rgbConverter, rightConverter;
```

Comment:
Any specific reason why unique ptr is created here ?

Reply:
The following parts have been followed. 
https://github.com/luxonis/depthai-ros-examples/blob/e9f2fd604f0cee291e30152be57fc7283f6f12c8/depthai_examples/ros1_src/stereo_nodelet.cpp#L21-L23
There is no particular reason to do otherwise. 
If it is unnecessary, we will correct it.

Reply:
I checked it works with OAK-D and OAK-D-LITE.
The unique_ptr was being used unnecessarily, so I changed it back.